.github/workflows/docker-publish.yml

```diff
@@ -27,10 +27,9 @@ jobs:
       # https://github.com/sigstore/cosign-installer
       - name: Install cosign
         if: github.event_name != 'pull_request'
-        uses: sigstore/cosign-installer@f3c664df7af409cb4873aa5068053ba9d61a57b6 #v2.6.0
-        with:
-          cosign-release: 'v1.13.1'
-
+        uses: sigstore/cosign-installer@v3.1.1
+      - name: Check install!
+        run: cosign version
```

Comment:
Is this intended to stay here? Since the above is restricted to non PRs, this should probably be too (if it is intentional)

Reply:
ahh good point - it was mostly for diagnostics